lang/alan/Portfile

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        alantech alan 0.1.7 v
+revision            0
+
+platforms           darwin
+categories          lang
+
+license             Apache-2 \
+                    AGPL-3
+
+homepage            https://alan-lang.org/
+
+description         The Alan Programming Language
+
+long_description    {*}${description}: The alan compiler and runtime can \
+                    parallelize your code without concurrent or asynchronous \
+                    programming \(threads, promises, channels, etc\) by only \
+                    allowing iteration and recursion that is guaranteed to \
+                    halt (e.g. no while (true) \{\} loops)
+
+checksums           rmd160  f66acbfb8e35272718d58c8c159d28d234ed8140 \
+                    sha256  c84fd2971bd820bcea659eae6400a50358722f46387510a1fe671c0201739280 \
+                    size    292965
+
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+depends_build       port:cargo \
+                    port:nodejs12 \
+                    port:python27 \
```

Comment:
Can python38 be used here instead? python 2.7 is not supported upstream and we are trying to move away from it.

Reply:
I know.  The project says it specifically wants Python 2.7: https://github.com/alantech/alan#source-installation
I haven't tried forcing use of python38 though...